### PR TITLE
Start v0.8.x

### PR DESCRIPTION
**Pull request by cdeszaq**
Based off of previous branch. Only last commit is relevant

### CODE REVIEW

**Comment by michael-mclawhorn** [comment]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,60 +8,104 @@ pull request if there was one.
...
+- Deadlock in `LuceneSearchProvider`
+- CORS support when using the `RoleBasedAuthFilter`
+
+New Capabilities & Enhancements:
+- Dimension field tagging
+- Controls around max size of Druid response to cache
+- Logging and timing enhancements
+
+Deprecations / Removals:
+- `RequestLog::switchTiming` is deprecated due to it's difficulty to use correctly
```
Comment (archolewa) [inline CHANGELOG.md]: `it's` should be `its`.

**Comment by archolewa** [comment]
One small grammar fix, but otherwise 👍

File: CHANGELOG.md (hunk 1 above)
Comment (cdeszaq) [inline CHANGELOG.md]: Too late, that commit already merged 😛

**Pull request merged**